h2o-parsers/h2o-parquet-parser/testMultiNode.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
unrelated comment: We need to get rid of this launcher or at least share them across projects.